support-models/src/main/scala/com/gu/support/workers/states/CreateZuoraSubscriptionState.scala

```diff
@@ -66,29 +64,14 @@ object CreateZuoraSubscriptionProductState {
       salesForceContact: SalesforceContactRecord,
   ) extends CreateZuoraSubscriptionProductState
 
-  case class DigitalSubscriptionDirectPurchaseState(
```

Comment:
This unwieldy name was to distinguish the state needed for a regular DS purchase from a gift purchase or a gift redemption. Now we only have regular purchases, I have simplified it.